### PR TITLE
HTML CSS Popup

### PR DESCRIPTION
**Pull request by nvklap**
### HTML CSS Popup

[Demo](https://nvklap.github.io/hooli-style-popup/) |
[Code base](https://github.com/nvklap/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by nvklap** [comment]
I checked the code and didn't find any common mistakes.

File: submissions/nvklap/popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  background-image: linear-gradient(19deg, #21d4fd 0%, #b721ff 100%);
+  border-radius: 5px;
+  margin: 0 auto;
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+}
+
+.logo img {
```
Comment (A-Ostrovnyy) [inline submissions/nvklap/popup/css/style.css:33]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Also, avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

File: submissions/nvklap/popup/index.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    <title>Hooli</title>
+    <link rel="shortcut icon" href="./img/icons/favicon.png" />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <a href="#" class="logo">
+          <img src="./img/pawww.png" alt="Logo" />
+        </a>
```
Comment (A-Ostrovnyy) [inline submissions/nvklap/popup/index.html:16]: :fire:

File: submissions/nvklap/popup/index.html [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+                      id="Outline"
+                      viewBox="0 0 24 24"
+                      width="27"
+                      height="27"
+                    >
+                      <path
+                        d="M18.71,8.21a1,1,0,0,0-1.42,0l-4.58,4.58a1,1,0,0,1-1.42,0L6.71,8.21a1,1,0,0,0-1.42,0,1,1,0,0,0,0,1.41l4.59,4.59a3,3,0,0,0,4.24,0l4.59-4.59A1,1,0,0,0,18.71,8.21Z"
+                      />
+                    </svg>
+                  </label>
```
Comment (A-Ostrovnyy) [inline submissions/nvklap/popup/index.html:142]: Logically when the second part of popup is open you should hide `More` button)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Sorry for the late response

**Comment by nvklap** [comment]
Thank you for the feedback. Provided articles have been very insightful. I made several fixes. Please, review my submission again.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**